src/Container.php

```diff
@@ -297,7 +297,8 @@ public static function create(): self
                 $config = $container->getConfiguration();
 
                 return new CoverageRequirementChecker(
-                    $config->getCoveragePath() !== '',
+                    $config->shouldSkipCoverage(),
```

Comment:
I noticed the wrong argument was being passed here so I corrected it, would you rather this was in a separate PR?